windows-mixin/dashboards/windows_logs.libsonnet

```diff
@@ -4,16 +4,17 @@ local custom_barchart_grafonnet = import '../lib/custom-barchart-grafonnet/custo
 
 local host_matcher = 'job=~"$job", agent_hostname=~"$hostname"';
 local log_channel_matcher = host_matcher + ', channel=~"$channel"';
+local windows_event_parser = '| json | line_format "{{.execution_processId}} Source: {{.source}} EventID: {{.event_id}} Level: {{.levelText}} . {{.message}}"';
```

Comment:
Two questions, just some thoughts:
1) WDYT about not using dot as separator?
2) I suggest to either add label to processId field or remove processId altogether. Since all extracted fields you don't use while composing line_format would be still available when you click on the line.

Reply:
@v-zhuravlev Generally in logs thread ID or process ID is usually present that is why I kept it there. I will add a label to it.

Also what do you suggest we use for separator instead of dot?

Reply:
just to be clear. By label here I meant 'text label' like 'Process id: {{.execution_processId}} '. Not Logs stream label.